ansible/configs/ocp4-ha-lab/default_vars.yml

```diff
@@ -159,8 +157,6 @@ instances:
     value: "utility"
   - key: "ostype"
     value: "linux"
-  - key: "Purpose"
-    value: "{{ purpose }}"
```

Comment:
Kind of.

I tested the open environment with an ocp4-ha-lab, and this key was breaking the provisioning; plus, it's useless because we set the cloud tags correctly in another place.